src/server.js

```diff
@@ -63,8 +63,8 @@ app.all('*', (req, res) =>
 );
 
 // finally, let's start our server...
-app.listen(process.env.PORT || 3000, () => {
-  console.log(`Listening on port`);
+const server = app.listen(process.env.PORT || 3000, () => {
```

Comment:
why create the `server` constant?

Reply:
It was in develop before now, my last PR removed it, I am just returning it back